build/media_source/mod_community_info/joomla.asset.json

```
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://developer.joomla.org/schemas/json-schema/web_assets.json",
+  "name": "mod_community_info",
+  "version": "1.0.0",
+  "description": "Assets for the communit info module",
+  "license": "GPL-2.0-or-later",
+  "assets": [
+    {
+      "name": "communityinfo.style",
+      "type": "style",
+      "uri": "mod_community_info/community-info.css"
+    },
+    {
+      "name": "communityinfo.script",
+      "type": "script",
+      "uri": "mod_community_info/community-info.es6.js",
```

```suggestion
      "uri": "mod_community_info/community-info.js",
```

The script in the media folder will be `community-info.js` the `community-info.es6.js` is the source file

---

Done. Thanks for your help.